docs/pagination.md

```diff
@@ -104,6 +104,26 @@ export default DRFSerializer.extend({
 });
 ```
 
+## Cursor Pagination
+
+To use `CursorPagination`, override `extractPageNumber` in the serializer to extract the `cursor`.
```

Comment:
:point_up: Can you link this to the DRF docs on cursor pagination?

http://www.django-rest-framework.org/api-guide/pagination/#cursorpagination